src/environment.ts

```diff
@@ -30,6 +30,8 @@ export enum Type {
 export interface Features {
   // Whether to enable debug mode.
   'DEBUG'?: boolean;
+  // Whether we are in a node.js (i.e., non-browser) environment.
+  'IS_NODE'?: boolean;
```

Comment:
This seems a little worrisome to me.   We have IS_NODE here and BACKEND below, shouldn't IS_NODE be derivable from BACKEND?

If this were a "List of features that current backend supports, then I would expect to not see the BACKEND token in this list, and IS_NODE might be ok.